apps/pay/components/sidebar/index.tsx

```diff
@@ -96,11 +101,27 @@ export function SideBar({ username }: { username: string }) {
             <span className="block w-8 h-0.5 bg-black"></span>
           </button>
         </SheetTrigger>
-        <SheetContent>
+        <SheetContent className="overflow-y-auto">
           <SheetHeader>
-            <p className="text-xl font-semibold text-left">Pay {username}</p>
+            <p className="text-xl font-semibold text-left m-0">Pay {username}</p>
           </SheetHeader>
           <div className="grid gap-3 py-3  ">
+            <div
+              className="flex flex-col gap-0 bg-slate-200 p-2 m-0 rounded-md"
+              onClick={() => {
+                if (!signedInUser) signIn("blink")
+              }}
+            >
+              {signedInUser ? (
+                <>
+                  <p className="text-md font-semibold mb-1">Signed in as</p>
+                  <p className="text-sm mb-0">{signedInUser.username}</p>
+                </>
+              ) : (
+                <p className="text-md font-semibold mb-1">Sign in</p>
+              )}
+            </div>
```

Comment:
we're also missing a log out / clear cache. not only for the case we are logged in, but also just to remove the username.

I had to change the username doing a demo this week, and I had to find the settings in chrome to clear all cache to be able to do it. we should have the option here.